lib/versions.js

```diff
@@ -6,6 +6,21 @@ import { HttpClient } from "@actions/http-client";
 export async function determineVersion(version, { repo, nightliesRepo }) {
   version = version?.trim();
 
+  const toolVersionsPathInput = core.getInput("tool-versions");
+
+  if (toolVersionsPathInput != "") {
+    let toolVersion = await getVersionFromToolVersionsFile(
+      toolVersionsPathInput,
+    );
+
+    if (!toolVersion) {
+      throw new Error(
+        `failed to determine version from tool-versions option, please double check the provided path`,
```

Comment:
I believe the inner `fs.readFile` will already yell at the user if the file is missing, but we get false-like value here if the file does exist, but misses `scarb *` line.

```suggestion
        `failed to read Scarb version from: ${toolVersionsPathInput}`,
```